ReSharper.FSharp/src/FSharp.Psi.Services/src/Daemon/Highlightings/FcsErrors.xml

```diff
@@ -560,4 +560,20 @@
     <QuickFix>RemoveXmlDocFix</QuickFix>
   </Warning>
 
+  <Error compilerGroup="FSharpErrors" name="SignatureFileMismatch" ID="FS0034: Signature file mismatch">
+    <Parameter type="IFSharpPattern" name="pattern"/>
+    <Message value="Signature file mismatch"/>
```

Comment:
This message doesn't explain the user what's wrong. We should use the FCS message in this case.